app/(pages)/welcome-page/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { useGetTest } from "@/app/api/hooks/test/useGetTest";
+import { Link } from "expo-router";
+import { useEffect } from "react";
+import { View, Text, Button } from "react-native";
+
+export default function WelcomePage() {
+  const { data, isLoading, isError } = useGetTest();
+  useEffect(() => {}, [data]);
+  return (
+    <View>
+      <Text>{data && data.title}</Text>
+      <Text> {isLoading && "Loading"}</Text>
+      <Text>{isError && "Error"}</Text>
```

Comment:
W przyszłości to pewnie zrobimy jakiegoś małego taba spadającego z tym errorem?

Reply:
Tak, sa na to tez libki, dodam na to potem taska w jirze, trzeba bedzie pomyslec czy chcemy na kazdy fail3d request tego taba pokazywac, bo jak tak to mozna zrobic custom hooka ktory owrapuje tamstack query, a w naszych hookach do zapytan juz bysmy tylko uzyeali tego wrappera